File: assign1/scala/src/test/scala/imperative_test.scala
class imperative_test extends perfect_number_test {
  
  test("Canary test") {
    assert(true)
  }

  test("Imperative: is perfect number") {
    testIsPerfectNumber(imperativePerfectNumber)
  }

  test("Imperative: is not perfect number") {
    testIsNotPerfectNumber(imperativePerfectNumber)
  }
}

File: assign1/go/test/imperative_test.go
package test 

import (
  "assign1/marin_yii/perfectnumber"
  "testing"
)

func TestImperative_is_perfect_number(t *testing.T) {
  test_is_perfect_number(t, perfectnumber.ImperativePerfectNumber)
}

func TestImperative_is_not_perfect_number(t *testing.T) {
  test_is_not_perfect_number(t, perfectnumber.ImperativePerfectNumber)
}
